app/models/user.rb

```diff
@@ -1,17 +1,29 @@
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
-  devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :validatable
+  devise :database_authenticatable, :recoverable, :rememberable, :validatable
   validates :name, presence: true
   validates :email, uniqueness: true
   has_many :memberships, dependent: :destroy
   has_many :roles, through: :memberships
-  has_one :organization, through: :memberships
+  # has one organization through memberships validations
+  has_many :organizations, through: :memberships
 
   def superuser?
     superuser = Role.find_by(name: 'superuser')
     memberships.find_by(role: superuser)
   end
 
+  def admin?
+    admin = Role.find_by(name: 'admin')
+    memberships.find_by(role: admin)
```

Comment:
hmmm I can get all of the users roles with `roles` query then there is no need to hit database twice.